packages/codegen-ui-react/lib/imports/import-mapping.ts

```diff
@@ -16,6 +16,7 @@
 export enum ImportSource {
   REACT = 'react',
   UI_REACT = '@aws-amplify/ui-react',
+  UI_REACT_TYPES = '@aws-amplify/ui-react/dist/types',
```

Comment:
do we need this path to import the types?

Reply:
Thanks, we dont need to. Will update